### DISCUSSION TITLE
Release periods

### DISCUSSION THREAD

**Discussion by xsmrcek** [open-telemetry/opentelemetry-java-instrumentation#5351] [Q&A]
Hello,
I would like to ask if there are any release periods, or how are releases scheduled.
If there is none, would it be possible to do a release because of this issue?
https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/5154
Thank you!

**Comment by laurit** [comment]
We do releases roughly once a month. [v1.11.0](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.11.0 ) was just released and includes that fix.

**Reply by xsmrcek** [reply]
Thank you! and I am sorry.

**Reply by xsmrcek** [reply]
I just retested it and it is working as expected! Thank you again.

**Comment by xsmrcek** [comment]
Hello @laurit , 
Can I have one more questiong according to propagation in my example?
![image](https://user-images.githubusercontent.com/50624143/153862472-2bf30f35-e98a-4b06-a638-f32770d710b0.png)
I am still missing MDC info in logs (propagation works fine). Do you think I can be another issue for agent?
(MDC info present in client request is there because I did some adjustment in there, but it is not suitable for anything else)

**Reply by laurit** [reply]
@xsmrcek It is hard to say, you'll have to provide a minimal sample app that reproduces the issue if you want someone to take a look. Besides bugs another reason why trace and span id might not be available through mdc is that perhaps this logging happens before context has been extracted from request.

**Reply by xsmrcek** [reply]
Thank you! I created another issue :)
